src/ert/libres_facade.py

```diff
@@ -285,10 +277,20 @@ def run_ertscript(  # type: ignore
         storage: Storage,
```

Comment:
Is the `run_ertscript` ever used, or can we get rid of it?

Reply:
Just used in semeio tests, so not sure what to do about it, would still like to be able to test them there.